compiler/modules/front/resolveTypes.bal

```diff
@@ -131,23 +131,32 @@ function resolveTypeDesc(ModuleSymbols mod, s:ModuleLevelDefn modDefn, int depth
     final t:Env env = mod.tc.env;
     if td is s:BinaryTypeDesc {
         // NB depth does not increase here
-        t:SemType l = check resolveTypeDesc(mod, modDefn, depth, td.left);
-        t:SemType r = check resolveTypeDesc(mod, modDefn, depth, td.right);
+        t:SemType accumilatorTy = check resolveTypeDesc(mod, modDefn, depth, td.operands[0]);
+        int nextIndex = 1;
```

Comment:
It's `accumulator` with a `u`. But this is a bad name because `accumulator` is unabbreviated but `ty` is abbreviated.  Use e.g. `accumType`.

Reply:
fixed